web/src/components/faq.js

```diff
@@ -72,6 +72,10 @@ const faq = {
         question: 'Do you take into account imports and exports of electricity?',
         answer: 'Yes, we do. Imports and exports can be see on the map as small arrows between different areas [link to question about arrows]. Detailed information can be seen in the charts shown when you click on an area.',
       },
+      emissionsOfStored: {
+        question: 'What about emissions generating electricity which is then stored in batteries or used to fill reservoirs?',
```

Comment:
can you change to "emissions from generating electricity"? I think that will read better.